src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,280 @@
+<!-- If using debug menu shortcuts, prompts player to enter their name before beginning initial Roach Market sequence -->
+::RM-first-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
```

Comment:
We could leave this in in the final game and just gate this all behind a `(if: (text: $playerName)'s length < 1)`, too, couldn't we?